constructor_io/modules/quizzes.py

```diff
@@ -0,0 +1,139 @@
+'''Quizzes Module'''
+
+from time import time
+from urllib.parse import quote, urlencode
+
+import requests as r
+
+from constructor_io.helpers.exception import ConstructorException
+from constructor_io.helpers.utils import (clean_params, create_auth_header,
+                                          create_request_headers,
+                                          create_shared_query_params,
+                                          throw_http_exception_from_response)
+
+
+def _create_quizzes_url(quiz_id, parameters, user_parameters, options, path):
+    # pylint: disable=too-many-branches
+    '''Create URL from supplied quiz_id and parameters'''
+    quiz_service_url = 'https://quizzes.cnstrc.com'
+    query_params = create_shared_query_params(options, {}, user_parameters)
+    ans_query_string = ''
+
+    if not quiz_id or not isinstance(quiz_id, str):
+        raise ConstructorException('quiz_id is a required parameter of type str')
+
+    if path == 'finalize' and (not isinstance(parameters.get('a'), list) or len(parameters.get('a')) == 0): # pylint: disable=line-too-long
+        raise ConstructorException('a is a required parameter of type list')
+
+    if parameters:
+        if parameters.get('section'):
+            query_params['section'] = parameters.get('section')
+
+        if parameters.get('version_id'):
```

Comment:
I just realized that `version_id` can be added as a parameter, but is this actually used? It's not specified in the [docs](https://docs.constructor.io/rest_api/quiz/using_quizzes/) as a parameter. Might be an artifact of the old docs? 🤔 

I think we allow it to be passed as a parameter in the other SDK repos as well... hm...

Reply:
It's specified here. It is supported I guess they just forgot to add it here.

https://dev-quizzes.cnstrc.com/docs/

Reply:
Cool beans! 😄